README.md

```diff
@@ -57,6 +57,9 @@ See instructions on the [GitHub release page](https://github.com/catppuccin/user
    - Replacing `<name>` with the name of the userstyle.
 3. Enjoy!
 
+> [!TIP]
+> For decentralized or self-hosted applications, you can apply a theme to one or multiple instances across updates by going to the userstyle's **Settings** > **Custom included sites** and entering URL patterns for your instance(s).
```

Comment:
```suggestion
> For decentralized or self-hosted applications, you can apply a theme to one or multiple instances across updates by following the instructions below.
> 1. Open the Stylus "Manage" page.
> 2. Click on the userstyle's name in the list.
> 3. Click on **Style settings** in the left panel.
> 4. Enter the URL patterns for your instance(s) in the text box labeled **Custom included sites**.
```

Rendered version below:
> [!TIP]
> For decentralized or self-hosted applications, you can apply a theme to one or multiple instances across updates by following the instructions below.
> 1. Open the Stylus "Manage" page.
> 2. Click on the userstyle's name in the list.
> 3. Click on **Style settings** in the left panel.
> 4. Enter the URL patterns for your instance(s) in the text box labeled **Custom included sites**.

Reply:
I had trouble even finding that text box at first since I was thinking gear icon = settings page, hopefully these instructions make more sense to the next person.

Reply:
Thats fair, its a fairly obscure feature.